neurobooth_os/tasks/s2s/task.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import, division
+import neurobooth_os
+import os.path as op
+from neurobooth_os.tasks.utils import make_win
+from neurobooth_os.tasks import utils
```

Comment:
these go below psychopy

Reply:
this is what pep8 says:

![image](https://user-images.githubusercontent.com/15852194/135568488-82670874-9356-4fd8-914f-b0b854af2ae5.png)